### ISSUE TITLE
Question about the dataset split

### ISSUE THREAD

**Issue by 1874Mercury** [luwei0917/TankBind#5]
Hi, it's really a great work!  But I'm a little confused about the PDBbindv2020 dataset split.
In the arxiv paper you wrote 

> We followed the same time split as defined in EquiBind paper ........we had 17787 structures for training, 968 for validation and 363 for testing

but the EquiBind paper wrote

>  From the remaining complexes that are older than 2019, we remove those with ligands contained in the test set, giving 17 347 complexes for training and validation. These are divided into 968 validation complexes, which share no ligands with the remaining 16 379 train complexes

so, compared with the EquiBind train dataset,  your train dataset contains 1408 more samples due to not remove those with same ligands contained in the test set?

If that's the case, I'm concerned this may lead to data leakage...

**Comment by luwei0917** [comment]
Hello, Thanks for the kind words.
This is a very good point, I will redo the training with those removed and get back to you. A quick check shows there are 48 smiles having been seen in the training set (14% of 363 total), I feel it won't affect the conclusion much.

**Comment by luwei0917** [comment]
Hello, Thanks for the kind words.
This is a very good point, I will redo the training with those removed and get back to you. A quick check shows there are 48 smiles having been seen in the training set (14% of 363 total), I feel it won't affect the conclusion much.

**Comment by 1874Mercury** [comment]
Thanks for the feedback! Looking forward to your next arXiv update！
